contracts/minter/src/contract.rs

```diff
@@ -621,6 +622,12 @@ fn query_mintable_num_tokens(deps: Deps) -> StdResult<MintableNumTokensResponse>
         count: count as u64,
     })
 }
+
+fn query_mint_price(deps: Deps) -> StdResult<MintPriceResponse> {
+    let mint_price = mint_price(deps, false)?;
+    Ok(MintPriceResponse { price: mint_price })
```

Comment:
should we add whitelist price if available?

Reply:
mint_price gives whitelist price if whitelist is active, otherwise gives mint price